x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/searchable_snapshot_field/searchable_snapshot_field.tsx

```diff
@@ -0,0 +1,231 @@
+            {
+              defaultMessage: 'Reload snapshot repositories',
+            }
+          )}
+          title={
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.editPolicy.loadSnapshotRepositoriesErrorTitle"
+              defaultMessage="Unable to load snapshot repositories"
+            />
+          }
+          body={
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.editPolicy.loadSnapshotRepositoriesErrorBody"
+              defaultMessage="Refresh this field and enter the name of an existing snapshot repository."
+            />
+          }
+        />
+      );
+    } else if (repos.length === 0) {
+      calloutContent = (
+        <EuiCallOut
+          data-test-subj="noSnapshotRepositoriesCallout"
+          iconType="help"
+          color="warning"
+          title={
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.editPolicy.noSnapshotRepositoriesFoundTitle"
+              defaultMessage="No snapshot repositories found"
+            />
+          }
+        >
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.noSnapshotRepositoriesFoundMessage"
+            defaultMessage="{link} to use searchable snapshots."
+            values={{
+              link: (
+                <EuiLink
+                  href={getUrlForApp('management', {
+                    path: `data/snapshot_restore/add_repository`,
+                  })}
+                  target="_blank"
+                >
+                  {i18n.translate(
+                    'xpack.indexLifecycleMgmt.editPolicy.createSearchableSnapshotLink',
+                    {
+                      defaultMessage: 'Create a snapshot repository',
+                    }
+                  )}
+                </EuiLink>
+              ),
+            }}
+          />
+        </EuiCallOut>
+      );
+    }
+  }
+
+  return (
+    <DescribedFormField
+      data-test-subj={`searchableSnapshotField-${phase}`}
+      switchProps={{
+        'data-test-subj': 'searchableSnapshotToggle',
+        label: i18n.translate(
+          'xpack.indexLifecycleMgmt.editPolicy.searchableSnapshotsToggleLabel',
+          { defaultMessage: 'Use searchable snapshot' }
+        ),
+        initialValue: Boolean(
+          policy.phases[phase]?.actions?.searchable_snapshot?.snapshot_repository
+        ),
+      }}
+      title={
+        <h3>
+          {i18n.translate('xpack.indexLifecycleMgmt.editPolicy.searchableSnapshotFieldTitle', {
+            defaultMessage: 'Searchable snapshot',
+          })}
+        </h3>
+      }
+      description={
+        <EuiTextColor color="subdued">
+          {i18n.translate(
+            'xpack.indexLifecycleMgmt.editPolicy.searchableSnapshotFieldDescription',
+            {
+              defaultMessage: 'Take a snapshot of the index and mount it as a searchable snapshot.',
```

Comment:
I think that a bit more context here is useful. 
>Take a snapshot of the managed index in the selected repository and mount it as a searchable snapshot. Learn more

Where **Learn more** links to the searchable snapshots page: https://github.com/elastic/elasticsearch/blob/master/docs/reference/ilm/actions/ilm-searchable-snapshot.asciidoc